x/wasmplus/keeper/test_common.go

```diff
@@ -688,3 +689,16 @@ func keyPubAddr() (crypto.PrivKey, crypto.PubKey, sdk.AccAddress) {
 	addr := sdk.AccAddress(pub.Address())
 	return key, pub, addr
 }
+
+func CreateOrderedAddresses(addrs ...[]byte) []string {
```

Comment:
"Create" and "Ordered" are a little ambiguous. How about naming `GenerateSortedBech32Address`?